packages/driver/src/cypress/log.ts

```diff
@@ -355,15 +355,7 @@ export class Log {
     this.set('snapshots', snapshots)
 
     if (options.next && shouldRebindSnapshotFn) {
-      const originalLogSnapshotFn = this.snapshot
-
-      this.snapshot = function () {
-        // restore the original snapshot function
-        this.snapshot = originalLogSnapshotFn
-
-        // call orig fn with next as name
-        return originalLogSnapshotFn.call(this, options.next)
-      }
```

Comment:
We need to maintain this for cross-origin snapshots.

Reply:
I merged in develop, which included a bunch of cross-origin changes, and fixed several merge conflicts; the tests were initially failing, but after some playing around I got them passing again.

Do you have a specific instance in mind where it's failing to snapshot cross-origin in this branch, so I can add a test?

Reply:
@AtofStryker ^^

Reply:
@emilyrohrbough @BlueWinds I don't believe there is an impact here to cross origin snapshots. 

In the case of asking the cross origin spec bridge for a snapshot, this should only happen in the case that something in the primary instance of cypress is requesting a snapshot while a `cy.origin` block is active or is about to be active, which means it's highly unlikely to be a command asking for the snapshot (ex: XHR's and fetch requests that ask for snapshots as a generated log that aren't direct command invocations). The tests in [snapshots.cy.ts](https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/e2e/e2e/origin/snapshots.cy.ts) should cover this for known cases. 

In the case were we DO need to rebind the original snapshot function, this usually happens on interaction events which should almost always be commands. Commands that are run within `cy.origin` take snapshots within the spec bridge itself, so no asking spec bridges for snapshots should be occuring. This should be covered by the various [command](https://github.com/cypress-io/cypress/tree/develop/packages/driver/cypress/e2e/e2e/origin/commands) tests that assert on `consoleProps` and other log attributes to verify the snapshot is correct.

Since we don't have tests that verify snapshots interactively (can't leverage the system test and cy-in-cy doesn't support snapshots), I ran this manually to double check and everything looks good from what I can see 😄 .

Reply:
Thank you for double checking! I'd hate to have accidentally broken something you merged in just a few days ago.

Reply:
Of course! Honestly wish we had a better way to test snapshots moving forward. Getting snapshot testing into `cy-in-cy` I think would cover a lot of ground.